core/src/main/scala/org/broadinstitute/dsde/rawls/util/RoleSupport.scala

```diff
@@ -28,20 +28,4 @@ trait RoleSupport {
           new RawlsExceptionWithErrorReport(errorReport = ErrorReport(StatusCodes.Forbidden, "You must be an admin."))
         )
     }
-
-  def tryIsCurator(userEmail: RawlsUserEmail): Future[Boolean] =
```

Comment:
This was easily eliminated by taking its one usage, and using `gcsDAO.isLibraryCurator` in the same `for` comprehension as the rest of the operations.